odoo/addons/base/data/res.country.state.csv

```diff
@@ -1735,3 +1735,15 @@ state_ke_44,ke,"Uasin Gishu",KE-44
 state_ke_45,ke,"Vihiga",KE-45
 state_ke_46,ke,"Wajir",KE-46
 state_ke_47,ke,"West Pokot",KE-47
+state_jo_1,base.jo,"Ajloun",JO-AJ
```

Comment:
Would be nice to try to keep some consistency
* try to order alphabetically by country
* use `jo` for the country xmlid

Is `JO-` really part of the state code?

Reply:
1. The file itself isn't alphabetically ordered. Plus, the last couple of commits that modified this file simply added at the end, which is why I did the same. Or did you mean order by city name?
2. Changed `base.jo` to `jo`
3. Yes, it would seem so. This is the ISO reference for city codes -  https://www.iso.org/obp/ui/#iso:code:3166:JO

Reply:
No, I meant to *try* to keep it sorted by country if possible but fair enough